project_event/models/project_task.py

```diff
@@ -265,15 +263,15 @@ def _compute_actual_total_time(self):
                         self.real_date_start))
                 hours = time_diff.hours
                 minutes = time_diff.minutes
-                self.total_time = "{0:0=2d}".format(
+                self.actual_total_time = "{0:0=2d}".format(
```

Comment:
The report also uses this field, you should refactor there too

Reply:
@eilst  Change in  report  project_task_activity_report file too